test/full.js

```diff
@@ -1542,74 +1542,31 @@ test('#68, #110 mixin props should not be deleted', function (t) {
     t.end();
 });
 
-test('#114 setOnce allows values to be set once and only once', function (t) {
```

Comment:
@jackboberg It looks like a few tests are being removed here. Is that intentional?